test/e2e/e2e_test.go

```diff
@@ -38,10 +38,10 @@ var (
 	catalogNamespace = flag.String(
 		"catalogNamespace", "", "namespace where the global catalog content is stored")
 
-	communityOperators = flag.String(
-		"communityOperators",
-		"quay.io/operatorhubio/catalog:latest",
-		"reference to upstream-community-operators image",
+	testCatalog = flag.String(
+		"testCatalog",
+		"quay.io/olmtest/test-catalog:latest",
```

Comment:
we use the latest tag of _our_ image (not the upstream image - i.e. it won't get pulled out from under us)
the intention here is not necessarily to solve the problem upstream (since it hasn't really been a problem), but rather: 
a) make it so that the image is a configurable parameter of the e2e (that way we can influence it downstream without having to backport changes) - so what this PR doesn't cover are the changes required to openshift/release
b) document the issue and lay the groundwork for a fix

I've already added a todo with an issue for building out a gha to update that image if necessary.
This helps me unblock the downstream. I can follow up some automation later. wdyt? or fix it here to v1.43.1?